subiquity/models/tests/test_filesystem.py

```diff
@@ -856,6 +856,8 @@ def fake_up_blockdata(model):
     for disk in model.all_disks():
         bd[disk.path] = {
             'DEVTYPE': 'disk',
+            'ID_SERIAL': disk.serial,
+            'ID_MODEL': disk.model,
```

Comment:
Did you want a separate disk where you match ID_VENDOR?  

Also, do we know what happens if multiple disks match? both VENDOR and MODEL are likely to hit duplicates in a multi-disk environment.

Reply:
> Did you want a separate disk where you match ID_VENDOR?

Good point, done.

> Also, do we know what happens if multiple disks match? both VENDOR and MODEL are likely to hit duplicates in a multi-disk environment.

This is documented! https://ubuntu.com/server/docs/install/autoinstall-reference -> 

    Any disk action is assigned a matching disk – chosen arbitrarily from the set of unassigned disks if there is more than one, and causing the installation to fail if there is no unassigned matching disk

and

    size: largest|smallest: take the largest or smallest disk rather than an arbitrary one if there are multiple matches (support for smallest added in version 20.06.1)